resources/sass/app.scss

```diff
@@ -3,3 +3,22 @@
 @tailwind components;
 
 @tailwind utilities;
+
+.status-insecure  {
+    --text-opacity: 1;
+    color: #c53030;
+    color: rgba(197, 48, 48, var(--text-opacity));
+    font-weight: 700;
+}
+
+.status-behind {
+    --text-opacity: 1;
+    color: #c53030;
+    color: rgba(197, 48, 48, var(--text-opacity));
+}
+
+.status-current {
+    --text-opacity: 1;
+    color: #2f855a;
+    color: rgba(47, 133, 90, var(--text-opacity));
+}
```

Comment:
Is it possible to avoid adding additional classes and instead use Tailwind classes on the element?

Reply:
Yep, is possible. How about this? https://github.com/tighten/checkmate/pull/31/commits/1173ec5ec9d4c12f4897d83fac1441b6df94ac0d